### DISCUSSION TITLE
Horizontal bar chart x and y label customized

### DISCUSSION THREAD

**Discussion by yasinguness** [Abhinandan-Kushwaha/react-native-gifted-charts#855] [Q&A]
I am trying to like belows image. 
![image](https://github.com/user-attachments/assets/7bc46df8-f93a-407c-8f61-5c088de9ff72)

Thats my current code. 
![image](https://github.com/user-attachments/assets/39916d86-8ab4-4c0b-a24e-7a30028da332)

Can i do like above images ? Is it possible ?

**Comment by Abhinandan-Kushwaha** [comment]
@yasinguness I would suggest you use a Population pyramid chart- https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/master/docs/PopulationPyramid/PopulationPyramid.md

**Reply by yasinguness** [reply]
But thats not same. I want to use just one values on same line. But population compare using two differenet values for same labels.

**Comment by yasinguness** [comment]
@Abhinandan-Kushwaha  is there any onPress action ? renderToolTip or pointerConfig like to another charts. Could you add like this action ?  Is it possible ?

**Comment by Abhinandan-Kushwaha** [comment]
@yasinguness `onPress` action is present in Pyramid charts, but not tooltip and pointerConfig.

**Reply by yasinguness** [reply]
Don't have onPress action. You should check

**Comment by Abhinandan-Kushwaha** [comment]
Hi @yasinguness 👋
I have added the props `onLeftPress` and `onRightPress` which accept a callback function with parameters item and index.
It is available from versions `1.4.43` onwards.

Please use the latest version of the library.
